assets/components/checkboxInput/checkboxInput.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  id: string,
```

Comment:
Do you want to add this to the default props as `null`, as @JustinPinner did for `errorMessage` in #237. Since we're only using this if label is present, do we need to pass it if we don't include a label?